Solutions/Menes.Hosting.AspNetCore/Menes/Internal/HttpRequestParameterBuilder.cs

```diff
@@ -313,6 +366,32 @@ private async Task TryAddBody(HttpRequest request, OpenApiOperationPathTemplate
                 return true;
             }
 
+            if (parameter.Schema.Default != null)
+            {
+                try
+                {
+                    result = this.TryGetDefaultValueFromSchema(parameter.Schema, parameter.Schema.Default);
+
+                    if (result != null)
+                    {
+                        this.logger.LogDebug(
+                            "Got default value for parameter [{parameter}] from the OpenAPI specification.",
+                            parameter.Name);
+
+                        return true;
+                    }
+                }
+                catch (OpenApiSpecificationException ex)
+                {
+                    this.logger.LogError(
+                        "Failed to parse default value for parameter [{parameter}] with [{schema}].",
+                        parameter.Name,
+                        parameter.Schema.GetLoggingInformation());
+
+                    throw ex;
```

Comment:
Are you intentionally using `throw ex;` rather than just `throw;`? When you re-throw an exception by specifying it in the `throw` statement, the original stack trace in the exception is lost, and it restarts from the current method which makes debugging harder.